app/assets/stylesheets/frontend/helpers/_featured-links.scss

```diff
@@ -1,12 +1,8 @@
 .featured-links {
   list-style: none;
 
-  @include media(tablet) {
-    padding-left: $gutter-two-thirds;
-  }
-
   li {
-    @include core-16;
+    @include core-19;
```

Comment:
Some poking around suggests that this `core-*` include is a whitehall-only feature, or even worse a feature from the previous govuk design system implementation 😱 Some poking around locally suggests as well that it's unnecessary. Could you instead try adding `govuk-link` to the parent `ul` and only using this selector to add the bold and play with the spacing? I'd also suggest checking out the components docs for lists as I think it has options for bold and adjusted spacing?

Reply:
Good spot, I didn't even clock, I thought this _was_ the Design System but of course it's prefixed differently.  I'll sort.